registration/registration-services/pom.xml

```diff
@@ -382,7 +382,7 @@
 		<dependency>
 			<artifactId>kernel-biosdk-provider</artifactId>
 			<groupId>io.mosip.kernel</groupId>
-			<version>${mosip.core.kernel.version}</version>
+			<version>${mosip.bio-utils.version}</version>
```

Comment:
Can't we use <mosip.biometric.util.version> property instead of creating new one.

Reply:
changed to mosip.biometric.util.version